src/ResourceManager/Network/Commands.Network/help/Get-AzureRmRouteFilterRuleConfig.md

```diff
@@ -18,16 +15,16 @@ Get-AzureRmRouteFilterRuleConfig [-Name <String>] -RouteFilter <PSRouteFilter>
 ```
 
 ## DESCRIPTION
-{{Fill in the Description}}
+The Get-AzureRmRouteFilterRuleConfig cmdlet is used to retrieve an existing route filter rule configuration associated to a route filter. The route filter rule configuration object returned can be used as input to other cmdlets that are related to route filters or route filter rule configs. A route filter is used in conjunction with Microsoft peering on an ExpressRoute circuit.
 
 ## EXAMPLES
 
-### Example 1
-```
-PS C:\> {{ Add example code here }}
+### Example 1: Get the route filter object and return the rule associated to the route filter
 ```
+$rf =  Get-AzureRmRouteFilter -Name "RouteFilter" -ResourceGroupName "ResourceGroupName"
```

Comment:
All commands should have the PS C:\> in front of them to format correctly.  Apply everywhere.